apps/assistants/migrations/0008_openaiassistant_is_archived_and_more.py

```
@@ -0,0 +1,29 @@
+            name='is_archived',
+            field=models.BooleanField(default=False),
+        ),
+        migrations.AddField(
+            model_name='openaiassistant',
+            name='version_number',
+            field=models.PositiveIntegerField(default=1),
+        ),
+        migrations.AddField(
+            model_name='openaiassistant',
+            name='working_version',
+            field=models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.CASCADE, related_name='versions', to='assistants.openaiassistant'),
```

Ruff

Line too long (162 > 120)

(E501)